charts/datahub/values.yaml

```diff
@@ -136,7 +136,7 @@ datahubUpgrade:
   enabled: true
   image:
     repository: acryldata/datahub-upgrade
-    tag: "v0.9.3"  # defaults to .global.datahub.version, v0.9.5 broken due to https://github.com/datahub-project/datahub/pull/6381
+    tag: "v0.9.5"  # defaults to .global.datahub.version, v0.9.5 broken due to https://github.com/datahub-project/datahub/pull/6381
```

Comment:
Should this have happened?

Reply:
Why

Reply:
The comment suggests that v0.9.5 is broken 🤷